### ISSUE TITLE
Suggestion: Warn when running on an old version of rubygems

### ISSUE THREAD

**Issue by jeremyolliver** [rubysec/bundler-audit#29]
On an older project, my ruby environment had rubygems version 1.6.2 installed which causes a false positive on non SemVer gem strings e.g. such as newrelic_rpm in cve 2013-0284 has a patched version of ">= 3.5.3.25", but the installed version of "3.5.5.38" doesn't satisfy the constraint. Updating rubygems (in my case to 1.8.19) and re-running fixes that, but I suggest printing a warning if the rubygems version is too old for that to work correctly. I'm not sure what the appropriate cutoff version would be.

TL;DR;

old rubygems versions can't compare gem versions with 4 period separated numbers correctly, so at least a warning would be appropriate
